### PR TITLE
:meat_on_bone: New SL logo lockup

### PR DESCRIPTION
**Pull request by ricardoom**
processed the new logo lockup and here is the svg version in this pr...

### CODE REVIEW

File: images/subwaylibrary.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 48" class="sl-logo" preserveAspectRatio="xMidYmid meet" aria-labelledby="subway-library">
+  <title id="subway-library">Subwary Library</title>
```
Comment (EdwinGuzman) [inline images/subwaylibrary.svg]: `subway-library-logo` is a better, more specific, id for this svg.

Comment (ricardoom) [inline images/subwaylibrary.svg]: true. i'm curious as to what screen readers actually read.... the `<title>` or the `aria` attributes...
I'll update

Comment (EdwinGuzman) [inline images/subwaylibrary.svg]: The `aria-label` which in this case will point to the title.

Comment (ricardoom) [inline images/subwaylibrary.svg]: but, i think we should use `aria-labelledby` which is the `id` of an element, in this case the `<title>`

**Comment by EdwinGuzman** [comment]
`aria-labelledby` is fine if the svg will only appear once on the page. The other svgs throw an error on the page and that must be fixed.

**Comment by EdwinGuzman** [comment]
Still think you should update it to `subway-library-logo`.

**Pull request merged**